src/modules/mount/main.py

```diff
@@ -321,6 +327,12 @@ def run():
         return (_("Configuration Error"),
                 _("No partitions are defined for <pre>{!s}</pre> to use.").format("mount"))
 
+    # swap
+    swap_partitions = [p['device'] for p in partitions if p['fs'] == 'linuxswap']
```

Comment:
You need to ensure the swap partition is "claimed".  We shouldn't enable an unclaimed partition.